app/upgrades/v20/constants.go

```diff
@@ -0,0 +1,11 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package v20
```

Comment:
this should be 19.1

Reply:
v19.1 was the previous one. But we can go with v19.2.0